kuma/users/templates/users/email/welcome/html.ltxt

```diff
@@ -56,6 +56,7 @@
             </li>
         </ul>
         <p>
+          {# Note: If you change this string also change ``WELCOME_EMAIL_STRINGS`` in kuma/users/tasks.py #}
```

Comment:
Yup, red herring. I took the `{% trans mdn_link=... %}` block out and the Spanish email worked. So I re-compiled the `es/LC_MESSAGES/django.po` file itself and that worked too.